fix(docs): content review (#2050)
scaleway/docs-content: tutorials/install-wireguard/index.mdx

    sudo apt install wireguard
    ```

> Once WireGuard is installed, you can check that the installation succeeded by running: `wg`, if you get no output it is all good. In order to check that the WireGuard kernel module has loaded you can run `sudo modprobe wireguard`.

Reviewer: ```suggestion
Once WireGuard is installed, you can check that the installation succeeded by running: `wg`. No output means that the installation has succeeded. In order to check that the WireGuard kernel module has loaded you can run `sudo modprobe wireguard`.
```
